src/sssom/cli.py

```diff
@@ -253,14 +252,18 @@ def split(input: str, output_directory: str):
 @input_argument
 @output_option
 @click.option("-W", "--inverse-factor", help="Inverse factor.")
-def ptable(input, output: TextIO, inverse_factor):
+@click.option(
+    "--default-confidence",
+    default=None,
```

Comment:
```suggestion
    default=None,
    type=click.FloatRange(0, 1),
```

Not sure if that works, maybe check

Reply:
I've changed it as per click documentation